internal/compiler/program.go

```diff
@@ -1553,11 +1595,23 @@ func (p *Program) GetIncludeReasons() map[tspath.Path][]*FileIncludeReason {
 
 // Testing only
 func (p *Program) IsMissingPath(path tspath.Path) bool {
-	return slices.ContainsFunc(p.missingFiles, func(missingPath string) bool {
+	return slices.ContainsFunc(p.missingFilePaths(), func(missingPath string) bool {
 		return p.toPath(missingPath) == path
```

Comment:
Couldn't this just be over missingPath still and then check `m.path` instead of us allocating a temp slice?